shared/src/main/scala/io/kaitai/struct/datatype/DataType.scala

```diff
@@ -163,6 +163,13 @@ object DataType {
     var args: Seq[Ast.expr]
   ) extends StructType {
     var classSpec: Option[ClassSpec] = None
+    /**
+     * Determines whether the user type represented by this `UserType` instance
+     * is external from the perspective of the given `ClassSpec` in which it is
+     * used (via `seq`, `instances` or `params`).
+     */
```

Comment:
Nit: I believe up until then the rest of our codebase uses docstrings formatted this way:

```suggestion
    /**
      * Determines whether the user type represented by this `UserType` instance
      * is external from the perspective of the given `ClassSpec` in which it is
      * used (via `seq`, `instances` or `params`).
      */
```

Reply:
Also ideally we should have `@param curClass` description here.